src/main/java/com/aws/greengrass/mqttbridge/MessageBridge.java

```diff
@@ -64,7 +70,6 @@ public MessageBridge(TopicMapping topicMapping) {
      */
     public void addOrReplaceMessageClient(TopicMapping.TopicType clientType, MessageClient messageClient) {
         messageClientMap.put(clientType, messageClient);
-        updateSubscriptionsForClient(clientType, messageClient);
```

Comment:
Trying to understand--looks like `updateSubscriptionsForClient` is now only called on startup for LocalMqtt, what's the reason for not needing it for PubSub & IotCore?

Reply:
Good catch, that was a miss. Fixed now. Thanks

Reply:
This is disturbing that testing did not find this. Can you add a test which will catch this issue?